.globalconfig

```diff
@@ -181,6 +181,9 @@ dotnet_diagnostic.CA1416.severity = warning
 # CA1417: Do not use 'OutAttribute' on string parameters for P/Invokes
 dotnet_diagnostic.CA1417.severity = warning
 
+# CA1418: Use valid platform string
+dotnet_diagnostic.CA1418.severity = silent
```

Comment:
I'd set warning on most of new rules if where no complains are raised and open new tracking issue for rest ones so that we review them late.

Reply:
Documentation for this rule does not yet exist, I think we should not yet enable it.

https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1418 -> 404

Reply:
It's temporary. 
It's important for us to make the rules work. If it is silent, then the rule does not work and there is no point in adding it. So I would put warning where possible and for other rules create a tracking issue so that enable them in follow PRs..